exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/DrillFunctionRegistry.java

```diff
@@ -122,6 +127,13 @@ public int size(){
   }
 
   public void register(DrillOperatorTable operatorTable) {
+    registerForInference(operatorTable);
+    registerForDefault(operatorTable);
```

Comment:
"Default" is not a good name (You choose "default", just because that's the existing behavior). Better to change to more meaningful name.